### PR TITLE
added keyword

### PR DESCRIPTION
**Pull request by DerThorsten**
not 100% what list of keywords to use.
I used this https://www.drupal.org/docs/develop/coding-standards/list-of-sql-reserved-words

### CODE REVIEW

File: src/xeus_sql_interpreter.cpp [hunk 1]
```diff
@@ -347,13 +356,879 @@ namespace xeus_sql
...
-
-    nl::json interpreter::complete_request_impl(const std::string& /*code*/,
-                                                int /*cursor_pos*/)
+    nl::json interpreter::complete_request_impl(const std::string& raw_code,
+                                                int cursor_pos)
     {
-        nl::json jresult;
-        jresult["status"] = "ok";
-        return jresult;
+        static const std::array<std::string, 826> keywords =
```
Comment (JohanMabille) [inline src/xeus_sql_interpreter.cpp:362]: Nitpicking: I think this array shoud be defined in a dedicated function at the end of the file, to improve readability (and avoid scrolling when you want to check what is defined below).

Comment (DerThorsten) [reply src/xeus_sql_interpreter.cpp:362]: we can return a const reference from the function since static const right?

Comment (DerThorsten) [reply src/xeus_sql_interpreter.cpp:474]: https://github.com/DerThorsten/xeus-sql/blob/32eb644c0b4303adb29ccce1e876d38d3f1ace4a/src/xeus_sql_interpreter.cpp#L472 for some reason the diff in this PR is not properly updated

Comment (JohanMabille) [reply src/xeus_sql_interpreter.cpp:474]: > we can return a const reference from the function since static const right?

Yes

Comment (marimeireles) [reply src/xeus_sql_interpreter.cpp:474]: Thank you! :)
I'll merge this :rainbow:

**Pull request merged**